app/components/molecules/TrackItem/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  ScrollView,
+  TouchableOpacity,
+  StyleSheet,
+  Text,
+  View,
+  Image
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 10,
+    marginTop: 3
+  },
+  item: {
+    padding: 10,
+    fontSize: 18,
+    height: 44
+  },
+  text: {
+    color: '#000000'
+  }
+});
+
+const state = {
+  tracks: [
+    {
+      id: 0,
+      trackName: 'Track 1'
+    },
+    {
+      id: 1,
+      trackName: 'Track 2'
+    },
+    {
+      id: 2,
+      trackName: 'Track 3'
+    },
+    {
+      id: 3,
+      trackName: 'Track 4'
+    }
+  ]
+};
```

Comment:
lets add a comment denoting :-

// for now we will be using hardcoded values, will update this in coming PR